### DISCUSSION TITLE
Using SID instead of storm's name

### DISCUSSION THREAD

**Discussion by pnathanoj** [umr-amap/StormR#99] [Ideas]
Some storms or cyclones have not been named (see the North Indian Basin or [the wiki page](https://en.wikipedia.org/wiki/List_of_unnamed_tropical_cyclones)) and are thus refered to as UNNAMED in the IBTrACS database. This is a problem when there are multiple unnamed storms within the same season.

Hence, do you considering using the storm identifier (SID) as an alternative to the storm name when it is unnamed (eg. for defStormsList)?

Many thanks!

**Comment by thomasarsouze** [comment]
Hi @pnathanoj,

thanks for your feedback. Indeed this is a behavior that needs to be improved.

I am wandering what is the best solution here: either the possibility to use the SID as an identifier or modify the name (using SID) so that we're sure that all storms have a unique name.

I'll check and test that ASAP and let you know.

**Comment by pnathanoj** [comment]
Thanks! I am currently using "names = "SID"" instead of "names = "NAME"" when creating the stormsDataset (the first solution that you have suggested). This seems to work flawlessly and allows me to bypass the issue with unnamed storms when running the spatialBehaviour function.
